mopidy_spotify/playlists.py

```diff
@@ -68,13 +100,96 @@ def refresh(self):
         self._loaded = True
 
     def create(self, name):
-        pass  # TODO
+        logger.info(f'Creating playlist {name}')
+        url = f'users/{web_client.user_id}/playlists'
+        response = self._backend._web_client.post(url, json={'name': name})
+        self.refresh()
+        return self.lookup(response['uri'])
 
     def delete(self, uri):
-        pass  # TODO
+        # Playlist deletion is not implemented in the web API, see
```

Comment:
there is an enpoint for it, but they called it 'unfollow': <https://developer.spotify.com/documentation/web-api/reference/follow/unfollow-playlist/>. [their rationale here](https://developer.spotify.com/documentation/general/guides/working-with-playlists/#following-and-unfollowing-a-playlist)